src/components/H1.js

```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+import { PALETTE } from "../shared";
+
+const H1 = styled.h1`
```

Comment:
I'd suggest using English names for H1/H2/H3, `<Header></Header>` is a lot more readable than `<H1></H1>`! It's more semantic, it decouples the name and the implementation, and it's easier to spot a mistyping.

Reply:
Hmm what would you name `H2`, `H3`?